evalml/tests/utils_tests/test_gen_utils.py

```diff
@@ -37,10 +37,14 @@ def in_container_arm64():
     The env var is set in the Dockerfile.arm for the purposes of local
     testing in a container on a mac M1, otherwise it's a noop.
     """
-    if os.getenv("DOCKER_ARM", None):
-        import plotly.io as pio
+    # if os.getenv("DOCKER_ARM", None):
+    import plotly.io as pio
 
-        pio.kaleido.scope.chromium_args += ("--single-process",)
+    pio.kaleido.scope.chromium_args += (
+        "--single-process",
+        "--headless",
+        "--disable-gpu",
+    )
```

Comment:
Originally these were only meant for mac m1, but windows also needs these settings for testing, else some tests hang forever, causing workers to fail.